gui/gui.go

```diff
@@ -734,6 +734,35 @@ func (gui *GUI) renderTerminalData(shouldLock bool) {
 			}
 		}
 	}
+	// hyperlinks
+	for y := 0; y < lineCount; y++ {
```

Comment:
Does it have sense to improve source code readability move hyperlinks rendering to separate function?